rest/class-boldgrid-backup-rest-archive.php

```diff
@@ -0,0 +1,277 @@
+<?php
+/**
+ * File: class-boldgrid-backup-rest-archive.php
+ *
+ * @link       https://www.boldgrid.com
+ * @since      X.X.X
```

Comment:
you're using X.X.X in this file instead of SINCEVERSION ..... Just putting this here for ya so these don't get missed when you do your search and replace later

Reply:
Thanks. https://github.com/BoldGrid/boldgrid-backup/commit/1b99ff2776d5ce9e883a4a87c9bf8153f29ad831